operator/chartdownloader/s3_test.go

```diff
@@ -27,52 +31,14 @@ func (m *mockS3) DownloadWithContext(ctx aws.Context, w io.WriterAt, input *s3.G
 	return int64(args.Int(0)), args.Error(1)
 }
 
-func TestDownloadSuccess(t *testing.T) {
```

Comment:
I removed these tests of the private `S3Downloader.download` method. It's called directly by the `S3Downloader.Download) method, which is tested.